projects/packages/forms/src/blocks/contact-form/components/jetpack-field-controls.js

```diff
@@ -163,105 +157,260 @@ const JetpackFieldControls = ( {
+						<>
+							<ToolsPanelItem
+								panelId={ clientId }
+								hasValue={ () => typeof attributes.buttonBorderWidth === 'number' }
+								label={ __( 'Button Border Width', 'jetpack-forms' ) }
+								onDeselect={ () =>
+									setAttributes( {
+										buttonBorderWidth: undefined,
+									} )
+								}
+							>
+								<RangeControl
+									label={ __( 'Button Border Width', 'jetpack-forms' ) }
+									value={ attributes.buttonBorderWidth }
+									initialPosition={ 1 }
+									onChange={ setNumberAttribute( 'buttonBorderWidth' ) }
+									min={ 0 }
+									max={ 100 }
+									__nextHasNoMarginBottom={ true }
+								/>
+							</ToolsPanelItem>
+							<ToolsPanelItem
+								panelId={ clientId }
+								hasValue={ () => typeof attributes.buttonBorderRadius === 'number' }
+								label={ __( 'Button Border Radius', 'jetpack-forms' ) }
+								onDeselect={ () =>
+									setAttributes( {
+										buttonBorderRadius: undefined,
+									} )
+								}
+							>
+								<RangeControl
+									label={ __( 'Button Border Radius', 'jetpack-forms' ) }
+									value={ attributes.buttonBorderRadius }
+									initialPosition={ 0 }
+									onChange={ setNumberAttribute( 'buttonBorderRadius' ) }
+									min={ 0 }
+									max={ 100 }
+									__nextHasNoMarginBottom={ true }
+								/>
+							</ToolsPanelItem>
+						</>
+					) }
+					{ ( ! isChoicesBlock || formStyle === FORM_STYLE.OUTLINED ) && (
+						<>
+							<ToolsPanelItem
+								panelId={ clientId }
+								hasValue={ () => typeof attributes.borderWidth === 'number' }
+								label={ __( 'Border Width', 'jetpack-forms' ) }
+								onDeselect={ () =>
+									setAttributes( {
+										borderWidth: undefined,
+									} )
+								}
+							>
+								<RangeControl
+									label={ __( 'Border Width', 'jetpack-forms' ) }
+									value={ attributes.borderWidth }
+									initialPosition={ 1 }
+									onChange={ setNumberAttribute( 'borderWidth' ) }
+									min={ 0 }
+									max={ 100 }
+									__nextHasNoMarginBottom={ true }
+								/>
+							</ToolsPanelItem>
+							<ToolsPanelItem
+								panelId={ clientId }
+								hasValue={ () => typeof attributes.borderRadius === 'number' }
+								label={ __( 'Border Radius', 'jetpack-forms' ) }
+								onDeselect={ () =>
+									setAttributes( {
+										borderRadius: undefined,
+									} )
+								}
+							>
+								<RangeControl
+									label={ __( 'Border Radius', 'jetpack-forms' ) }
+									value={ attributes.borderRadius }
+									initialPosition={ 0 }
+									onChange={ setNumberAttribute( 'borderRadius' ) }
+									min={ 0 }
+									max={ 100 }
+									__nextHasNoMarginBottom={ true }
+								/>
+							</ToolsPanelItem>
+						</>
+					) }
+				</ToolsPanel>
+				<ToolsPanel>
+					<div style={ { gridColumn: '1 / -1' } }>
+						<ToggleControl
+							label={ __( 'Sync field style', 'jetpack-forms' ) }
```

Comment:
I like the move to a singular `field` within the label, however, I'm less sold on `style` instead of `styles`. 

I understand it refers to the "overall style" of this field. That might not be immediately clear to end users however and without a greater separation from the single style above it, i.e. "border", it could cause some small confusion. 

What about tweaking this to `Sync field styles`? This would align more with core labels like "Copy styles", "Paste styles" etc. which refer to a collection of styles for a single block (see the paragraph block's more block toolbar menu).